mungegithub/README.md

```diff
@@ -11,7 +11,7 @@ Mungegithub is a tool based on GitHub that offers a better way to review and app
 Mungegithub requires a behavior change to use new system while doesn't block you to keep the old way, which although is not recommended.
 
 The core part of mungegithub, submit-queue automatically rebases, verities and merges PRs once they are approved.
-It waits for all reuqired tests passed, 4 kinds of labels(cla, lgtm, approve, release-note) before putting this pr into the queue. When a mergable pr comes to the head of Submit-queue, required retest contexts will be triggered to make sure everything is well tested. After these second-round tests pass, Mungegithub will go and merge this pr for you!
+It waits for all reuqired tests passed, 4 kinds of labels(cla, lgtm, approve, release-note) before putting this PR into the queue. When a mergable PR comes to the head of Submit-queue, required retest contexts will be triggered to make sure everything is well tested. After these second-round tests pass, Mungegithub will go and merge this PR for you!
```

Comment:
s/reuqired/required

It waits for all required tests to pass, and ensure PR has these four labels added - cla, lgtm, approve, release-note - before putting this PR into the queue. When a mergeable PR comes to the head of Submit-queue, it will trigger required retest contexts, to make sure everything is well tested. After these second-round tests pass, Mungegithub will go and merge this PR for you!